CDCgov/prime-reportstream
# Bg 2447 setting clia for sender FIX

This PR fixes the cliaForSender functionality by adding a call to the database. That call is required to collect the report ancestor and query the results metadata table to select the sender ID. Included in the PR is a change to refactor the cliaForSender code out of the createMessage function and into a dedicated function. Additionally, the unit test has been edited to test that new function explicitly. 

Test Steps:
1. pull CSV file from SimpleReport out of blobstorage
2. edit file to set patient state to CA
3. Set cliaForSender on california with "image_mover.default" and new CLIA (ex CDPH000075)
4. start docker (docker-compose up)
5. curl CSV file into docker
6. check that MSH-4-2 has been set to the new CLIA

## Changes
- moved cliaForSender to separate function
- added jooq query to retreive data from covid result metadata
- changed unit test to explicitly test cliaForSender function

## Checklist

### Testing
- [X] unit test 'test cliaForSender'

## Fixes
- #2447
